### ISSUE TITLE
Create Wiki or Manual

### ISSUE THREAD

**Issue by Boia11** [laurb9/StepperDriver#82]
Hi @laurb9 , thanks for your job.
Can you create wiki or manual for this libraries with list of function and how use it?
Please

**Comment by FreddyJohn** [comment]
Agreed! Please do this. It is easier to create my own program than use this library

**Comment by underwoodblog** [comment]
Yes, there are some features unclear that are not shown or described in the examples!
